config/locales/en.yml

```diff
@@ -253,6 +253,7 @@ en:
         ban_user: "Ban user"
         saving_user_status: Saving...
         save_user_status_successful: Saved
+        export_as_csv: "Export as CSV"
```

Comment:
This link doesn't export exactly the same content that is shown on the screen because we export all and ignore pagination and sorting. How about saying "Export all as CSV" here to make it clear?